homeassistant/components/tplink_lte.py

```diff
@@ -0,0 +1,103 @@
+"""
+Support for TP-Link LTE modems.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tplink_lte/
+"""
+import asyncio
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
```

Comment:
Please sort 🔡 within groups standard library, 3rd party and homeassistant imports.